advocacy_docs/edb-postgres-ai/analytics/external_tables.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Querying Delta tables in S3-compatible object storage
+navTitle: External Tables
+description: Access and Query data in S3-compatible object storage using External Tables
+deepToC: true
+---
+
+## Overview
+
+External tables allow you to access and query data stored in S3-compatible object storage using SQL. You can create an external table that references data in S3-compatible object storage and query the data using standard SQL commands.
```

Comment:
Technically we also support Azure and local filesystem (if you pass `file:///var/lib/aidb/data` as the storage location URL)

Reply:
Though if this is on BigAnimal, the FS access isn't useful, only if the user is installing the `pgaa` extension locally